mongomock/collection.py

```diff
@@ -13,8 +13,10 @@
 
 try:
     from bson import json_util, SON, BSON
+    from bson.codec_options import CodecOptions
 except ImportError:
     json_utils = SON = BSON = None
+    CodecOptions = type(None)
```

Comment:
If it's possible, I'd rather have it to be `None`, and that you check that it's not None before using it.